### ISSUE TITLE
Prism Launcher stuck on Initializing when launching an instance

### ISSUE THREAD

**Issue by Blayung** [PrismLauncher/PrismLauncher#3111]
### Operating System

Linux

### Version of Prism Launcher

9.1

### Version of Qt

5.12.8

### Description of bug

After a recent update of prismlauncher-qt5-bin on arch, all the data has moved from `~/.local/share/PrismLauncher` to `/opt/prismlauncher`. I've deleted all my cache when moving the instances and settings.

Now when I try to play, it just gets stuck on this:

![](https://github.com/user-attachments/assets/613c44d7-27cd-446f-a4f6-3624dfc5587e)

`30.292 D | [launcher.task.net.metacache.http]: Saving metacache with 14 entries`

Full logs: https://pastebin.com/jpNCCa60

### Steps to reproduce

Install prismlauncher-qt5-bin from the aur, try playing the game.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Blayung** [comment]
Okay, I've deleted `metacache`, `meta` and `accounts.json` and it worked. But I still do not know if this is intended behaviour, so I'm just gonna leave it.

**Comment by Blayung** [comment]
Nvm, it has worked only once, it's still broken.

**Comment by Blayung** [comment]
I have fixed it by changing the package to `prismlauncher`. I don't care anymore :D

**Comment by InvitedRhino683** [comment]
ive had this exact problem with the qt5 portable version, on kubuntu 24.04 lts. using the normal (qt6) portable version fixes it

**Comment by mineLdiver** [comment]
Having this issue on Arch Linux with the latest Prism Launcher on flathub. It does launch eventually, but sits there for a very long time (maybe a whole minute or something). Apart from it, opening secondary windows also exhibits the same lag.

**Comment by dimaguy** [comment]
I have this same issue on windows, with the added bonus of not being able to edit instance, and all the sub-windows opening and closing when closing prismlauncher

**Comment by payolin** [comment]
I had this issue too. Turns out I was using the 1.1.1.1 DNS server instead of the automatic one from my router. Changing it back to automatic fixed the problem.

**Comment by Torphedo** [comment]
> ive had this exact problem with the qt5 portable version, on kubuntu 24.04 lts. using the normal (qt6) portable version fixes it

> I have fixed it by changing the package to `prismlauncher`. I don't care anymore :D

Same problem and fix here.

**Comment by yash-garg** [comment]
@dimaguy were you able to get it working? I am facing the same issue!

**Comment by dimaguy** [comment]
> @dimaguy were you able to get it working? I am facing the same issue!

Nope, all I can do is reboot when it happens and hope it doesnt happen again

**Comment by grayscale420** [comment]
Also experiencing the same issue on Debian 12.10, Prism Launcher version 9.4

Edit:  Seemingly resolved by removing and re-adding Microsoft account.

**Comment by praecipitator** [comment]
Yep, had this issue as well. However, deleting accounts.json fixed it for me. I think the actual issue is that Prismlauncher sometimes isn't properly recognizing that it has bad account data.

**Comment by jleonardfl** [comment]
I am experiencing this issue with the Prism Launcher 9.4 flatpak on bazzite. I just tried deleting accounts.json and re-authenticating, but attempting to launch any instance stalls at 0%.

Version Info
```
Ref: app/org.prismlauncher.PrismLauncher/x86_64/stable
Arch: x86_64
Version: 9.4
Origin: flathub
Installation: system
Runtime: org.kde.Platform/x86_64/6.9
Sdk: org.kde.Sdk/x86_64/6.9

Commit: e2cd36b1b861c7c2d9f4f480ad9fd9d9e80f585427c4ef13cb191990a7b5a03f
Parent: b4ac0f1fc32b236f747246283e993208bfafbabce837727506a9ebfdbfd982a0
Subject: Merge pull request #92 from caesarw/master (fcd4fc73c502)
Date: 2025-05-18 03:00:21 +0000
```

**Comment by dominilces** [comment]
hello to anyone that is having this problem nowadays, the best solution, at least for arch users, is to use the Qt5 portable version from their website. For me, none of the solutions listed here worked at all, so thats the only solution i brought, hope that helps.

**Comment by Zrnik** [comment]
Not sure if i can help, but i had this problem too for last few days. My issue was related to https://github.com/PrismLauncher/PrismLauncher/issues/2074 and had same symptomps like in the description.

I am StarLink user, and my problem was IPv6. The moment i compeltely disabled IPv6 on windows level, the problems were gone (as IPv6 adresses stopped showing up in launch logs).

**Comment by rory-jenns** [comment]
Had this too. Worked after I disabled my VPN.

**Comment by pranavtaysheti** [comment]
Still experiencing same issue, I dont use VPN , but I have pi-hole set up on my network can anyone tell which domain I must allow?

**Comment by spencerrecneps** [comment]
I'm experiencing this with a pi-hole setup as well. Was working fine previously. Would love to know how to configure to pi-hole with whatever whitelist is needed, etc.

**Comment by SourLemonJuice** [comment]
I've also encountered this because I changed the Microsoft account password. It logged out the prism launcher's session. I can't refresh that account in the account manager at that time.

**Comment by Blayung** [comment]
Okay, this issue is so goddamn old, why isn't this fixed yet???

**Comment by michaellinhq** [comment]
Hi, I reviewed 'Prism Launcher stuck on Initializing when launching an instance' and can start immediately. I can deliver a first usable version quickly and iterate based on your feedback. If this scope still matches your priority, I can proceed right away.

**Comment by SourLemonJuice** [comment]
> Okay, this issue is so goddamn old, why isn't this fixed yet???

I think it's a network or account issue not a bug. But the message is indeed not clear.

**Comment by pranavtaysheti** [comment]
Last month I upgraded my homelab and moved my Pi-Hole from my aging Raspberry Pi to Intel NUC.  this issue has not occurred since then. I am using the same lists as my old setup. so, it can't be that I unblocked any address that was blocked before.

I believe, sometimes my Pi-Hole was too slow to respond, which was causing the issue.
